api/app/schemas.py

```diff
@@ -262,11 +263,11 @@ class ApplyInvitationRequest(ORMModel):
 
 class ActionLogResponse(ORMModel):
     logging_id: UUID
-    action_id: UUID
+    action_id: Optional[UUID] = None
```

Comment:
周りに合わせて、Optionalではなく、「|」で書きましょう。
```suggestion
    action_id: UUID | None = None
```
ActionLogRequestも同様

Reply:
修正しました。